src/main/java/com/softserve/academy/spaced/repetition/controller/CardRatingController.java

```diff
@@ -21,7 +21,7 @@
 import static org.springframework.hateoas.mvc.ControllerLinkBuilder.methodOn;
 
 @RestController
-@RequestMapping("api/decks/{deckId}/cards/{cardId}/rate")
+@RequestMapping("api/decks/cards/{cardId}/rate")
```

Comment:
Why changing the url? What is the impact on the UI part?

Reply:
DeckId never used. There is no impact on the UI, because rating for Cards not implemented on UI